*: update docs-dm (#643)
pingcap/docs-dm: en/deploy-a-dm-cluster-using-binary.md

      ./bin/dm-master -config conf/dm-master1.toml
      ```

      > **Note:**
      >

>       > After executing this command, the console will not output the log, you can view the running log through `tail -f dm-master.log`

Reviewer: ```suggestion
      > The console does not output logs after this command is executed. If you want to view the runtime log, you can execute `tail -f dm-master.log`.
```
